### PR TITLE
LPS-182766 | Add feedback to the user when moving an element using Move Modal in Knoledge Base

### PR DESCRIPTION
**Pull request by jaimelr10**
**Resend from**: https://github.com/liferay-lima/liferay-portal/pull/4477

**Description**: We need to provide the user a way to identify which and where is moving an element when using the new move modal for knowledge base.

**Jira Ticket**: https://issues.liferay.com/browse/LPS-182766

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java [hunk 1]
```diff
@@ -131,16 +131,16 @@ public JSONArray getKBFolderDataJSONArray() throws PortalException {
 			));
 	}
 
-	public long getKBObjectToMoveParentId() throws PortalException {
-		String kbObjectToMoveClassName = ParamUtil.getString(
-			_httpServletRequest, "kbObjectToMoveClassName");
+	public long getMoveKbObjectParentId() throws PortalException {
+		String moveKbObjectClassName = ParamUtil.getString(
```
Comment (ambrinchaudhary) [inline modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java:135]: the "b" should be uppercase: `getMoveKBObjectParentId()`, `moveKBObjectClassName`

**Review by ambrinchaudhary — CHANGES_REQUESTED** [review]
Just a minor change @jaimelr10 💪

**Comment by jaimelr10** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:forward:force

**Comment by Jeremy-Ch** [comment]
ci:forward

**Comment by Jeremy-Ch** [comment]
Send to BChan directly: https://github.com/brianchandotcom/liferay-portal/pull/138451

**Pull request closed without merging**